Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -421,6 +421,150 @@ public IEnumerator NearInteractionTouchableUnityUiToggle()
+            button.transform.SetParent(touchableUI.transform, false);
+            var catcherUI = new UnityButtonEventCatcher(button);
+
+            yield return new WaitForFixedUpdate();
+            yield return null;
+
+            touchableRect.gameObject.name = "Rect Touchable";
+            touchableVolume.gameObject.name = "Volume Touchable";
+            touchableUI.gameObject.name = "UnityUI Touchable";
+
+            float distSurface = 0.3f;
+            touchableRect.transform.position = new Vector3(-1, 0, distSurface);
+            touchableVolume.transform.position = new Vector3(0, 0, distSurface);
+            touchableUI.transform.position = new Vector3(1, 0, distSurface);
+
+            yield return TestTouchableDistances(touchableRect, 0.0075f, touchableRect.gameObject);
+            yield return TestTouchableDistances(touchableVolume, 0.0075f, touchableVolume.gameObject);
+            yield return TestTouchableDistances(touchableUI, 0.0f, button.gameObject);
+
+            UnityEngine.Object.Destroy(touchableRect.gameObject);
+            UnityEngine.Object.Destroy(touchableVolume.gameObject);
+            UnityEngine.Object.Destroy(touchableUI.gameObject);
+        }
+
+        private IEnumerator TestTouchableDistances(BaseNearInteractionTouchable touchable, float colliderThickness, GameObject objectDownExpected)
+        {
+            Handedness handedness = Handedness.Right;
+            ArticulatedHandPose.GestureId gesture = ArticulatedHandPose.GestureId.Open;
+
+            yield return PlayModeTestUtilities.ShowHand(handedness, inputSim);
+
+            PokePointer pokePointer = null;
+            var hand = HandJointUtils.FindHand(handedness);
+            Assert.IsNotNull(hand);
+            foreach (var pointer in hand.InputSource.Pointers)
+            {
+                pokePointer = pointer as PokePointer;
+                if (pokePointer)
+                {
+                    break;
+                }
+            }
+            Assert.IsNotNull(pokePointer);
+            float touchableDistance = pokePointer.TouchableDistance;
+
+            float pokeThreshold = 0.14f;
+            float debounceThreshold = 0.01f;
+            touchable.PokeThreshold = pokeThreshold;
+            touchable.DebounceThreshold = debounceThreshold;
+
+            Vector3 center = touchable.transform.position;
+
+            float margin = 0.001f;
+            Vector3 pStart = center + new Vector3(0, 0, -touchableDistance - 0.5f);
+            Vector3 pTouch = center + new Vector3(0, 0, -touchableDistance + margin);
+            // XXX the PokeThreshold ray start offset is not taken into account (see remarks above)
+            Vector3 pPoke = center + new Vector3(0, 0, -colliderThickness);
+            Vector3 pDebounce = center + new Vector3(0, 0, -colliderThickness - touchable.DebounceThreshold - margin);
+            //Vector3 pPoke = center + new Vector3(0, 0, -colliderThickness[i] - touchable.PokeThreshold + margin);
```

Comment:
Remove commented code

Reply:
I've left it here to show what behavior i would have expected: i thought the touch would start at `PokeDistance` - but it only starts when getting right up to the collider surface. I would like to get some confirmation (@julenka, @wiwei?) that this is the intended behavior and not a bug before removing this code.

Reply:
Ah I see, no worries

Reply:
@julenka would know more in this area than I would.

Reply:
The expected behavior is to not need to move right up to the collider unless PokeThreshold is zero. If PokeThreshold was 0.1, then the touch should start at distance 0.1 away.

Let's file a new bug and try to get this fixed soon. Not a must have for GA, but perhaps an "on deck" bug feature for the next release.